keyboards/keychron/k8/rgb/optical_iso/rules.mk

```diff
@@ -0,0 +1 @@
+OPT_DEFS += -DOPTICAL_MATRIX
```

Comment:
need to figure out what will happen with this. we need something that works against the main qmk scan now

Reply:
I left it here, as I was unsure and do not have an optical keyboard..

Reply:
Probably going to go with https://github.com/SonixQMK/qmk_firmware/pull/299 once it gets the green light from qmk

Reply:
this has now been updated in https://github.com/qmk/qmk_firmware/pull/19053
code is in upstream now, see the PR for usage refs